source/wp-content/themes/wporg-developer-2023/src/code-title/style.scss

```diff
@@ -1 +1,39 @@
 /* css styles */
+.wp-block-wporg-code-reference-title {
+	> * {
+		font-size: var(--wp--preset--font-size--extra-large);
+		line-height: 42px;
+		color: #008a20;
```

Comment:
I can see these colors are also used for code examples; is it worth adding presets for them, or sharing some other way so that we only have to change them in one place?

Reply:
I agree these should be defined somewhere specific.
We can't add presets to `theme.json` without duplicating all the existing values, but we could sorta fake it in the CSS by creating some new custom properties. The only drawback is that they won't be available in the editor itself.